libs/vkd3d/vkd3d_private.h

```diff
@@ -1807,6 +1807,8 @@ struct d3d12_graphics_pipeline_state
 
     VkPipelineLayout pipeline_layout;
     VkPipeline pipeline;
+    VkPipeline library;
```

Comment:
This seems to be unused?

Reply:
It's used in a later commit, I'll move the change.